tika-parsers/src/main/java/org/apache/tika/parser/html/BoilerpipeContentHandler.java

```diff
@@ -17,11 +17,9 @@
 package org.apache.tika.parser.html;
 
 import java.io.Writer;
-import java.util.ArrayList;
-import java.util.BitSet;
-import java.util.List;
-import java.util.Locale;
+import java.util.*;
```

Comment:
In general we try to avoid importing `.*` (other than `Assert.*`), though I see a few of these have slipped into the code in other places.

Reply:
Sure, I will make the changes.